libs/design-system/table/src/lib/table-demo/table-demo.component.html

```diff
@@ -0,0 +1,13 @@
+<table mat-table [dataSource]="dataSource" matSort aria-label="Table with sort function">
+  @for (column of columns; track column) {
```

Comment:
Just create one column def for `name` and `value` respectively instead of a for loop